package.json

```diff
@@ -266,25 +266,24 @@
     "vscode:prepublish": "yarn compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "yarn compile",
     "build": "vsce package --yarn",
     "vscode:publish": "vsce publish --yarn",
     "format": "prettier --write **/*.{ts,js,json,yml}",
     "format-check": "prettier --check **/*.{ts,js,json,yml}"
   },
   "devDependencies": {
-    "@types/node": "^13",
+    "@types/node": "^13.9.3",
```

Comment:
since we’re at it, can you remove all the ^ from the dependencies? They’re not very useful since we’re using yarn.lock and dependabot